src/profile-logic/comparison.js

```diff
@@ -163,6 +214,18 @@ type TranslationMapForCategories = Map<
   IndexIntoCategoryList,
   IndexIntoCategoryList
 >;
+type TranslationMapForFuncs = Map<IndexIntoFuncTable, IndexIntoFuncTable>;
```

Comment:
Optional: In other places in the codebase we've used names like `oldFuncToNewFuncMap`. I'm not opposed to this new naming style, but consistency can be nice.

https://github.com/firefox-devtools/profiler/search?q=oldFunc&unscoped_q=oldFunc

Reply:
I considered it but then decided to keep the names as is, mostly because this makes it easy and natural to use shorter names where possible (`translationMaps` vs `translationMapsForLibs`). This makes it nicer to use in this piece of code where we have a lot of these objects.